data/strings/types_strings.txt

```diff
@@ -9193,6 +9193,318 @@
+
+  [type.healthcare.optometrist]
+    en = Optometrist
+    ar = طبيب العيون
+    be = Акуліст
+    bg = Оптометрист
+    cs = Optometrista
+    da = Optometrist
+    de = Augenoptiker
+    el = Οπτομέτρης
+    es = Optometrista
+    et = Optometrist
+    eu = Optometrista
+    fa = اپتومتریست
+    fi = Optometristi
+    fr = Optométriste
+    he = אופטומטריסט
+    hu = Optometrista
+    id = Ahli kacamata
+    it = Optometrista
+    ja = オプトメトリスト
+    ko = 검안사
+    mr = ऑप्टोमेट्रिस्ट
+    nb = Optometrist
+    nl = Optometrist
+    pl = Optometrysta
+    pt = Optometrista
+    pt-BR = Optometrista
+    ro = Optometrist
+    ru = Оптометрист
+    sk = Optometrista
+    sv = Optometrist
+    sw = Daktari wa macho
+    th = นักทัศนมาตร
+    tr = Optometrist
+    uk = Оптометрист
+    vi = bác sĩ nhãn khoa
+    zh-Hans = 验光师
+    zh-Hant = 驗光師
+
+  [type.healthcare.podiatrist]
+    en = Podiatrist
+    ar = طبيب الأرجل
+    be = Ортопед
+    bg = Подиатрист
+    cs = Podiatr
+    da = Fodterapeut
+    de = Podologe
+    el = Ποδίατρος
+    es = Podólogo
+    et = Jalaravi
+    eu = Podologoa
+    fa = متخصص پا
+    fi = Jalkaterapeutti
+    fr = Podologue
+    he = פודיאטר
+    hu = Podiatrist
+    id = Ahli penyakit kaki
+    it = Podologo
+    ja = 足病医
+    ko = 발 전문의
+    mr = पोडियाट्रिस्ट
+    nb = Fotterapeut
+    nl = Podotherapeut
+    pl = Podolog
+    pt = Podologista
+    pt-BR = Podólogo
+    ro = Podiatru
+    ru = Врач-ортопед
+    sk = Podológ
+    sv = Fotvårdare
+    sw = Daktari wa miguu
+    th = แพทย์โรคเท้า
+    tr = Podiatrist
+    uk = Ортопед
+    vi = Bác sĩ chân
+    zh-Hans = 足科医生
+    zh-Hant = 足病醫生
+
+  [type.healthcare.psychotherapist]
+    en = Psychotherapist
+    ar = معالج نفسي
+    be = Псіхатэрапеўт
+    bg = Психотерапевт
+    cs = Psychoterapeut
+    da = Psykoterapeut
+    de = Psychotherapeutin
+    el = Ψυχοθεραπευτής
+    es = Psicoterapeuta
+    et = Psühhoterapeut
+    eu = Psikoterapeuta
+    fa = روان درمانگر
+    fi = Psykoterapeutti
+    fr = Psychothérapeute
+    he = פסיכותרפיסט
+    hu = Pszichoterapeuta
+    id = Psikoterapis
+    it = Psicoterapeuta
+    ja = サイコセラピスト
+    ko = 심리 치료사
+    mr = मानसोपचारतज्ज्ञ
+    nb = Psykoterapeut
+    nl = Psychotherapeut
+    pl = Psychoterapeuta
+    pt = Psicoterapeuta
+    pt-BR = Psicoterapeuta
+    ro = Psihoterapeut
+    ru = Психотерапевт
+    sk = Psychoterapeut
+    sv = Psykoterapeut
+    sw = Mwanasaikolojia
+    th = นักจิตบำบัด
+    tr = Psikoterapist
+    uk = Психотерапевт
+    vi = nhà trị liệu tâm lý
+    zh-Hans = 心理治疗师
+    zh-Hant = 心理治療師
+
+  [type.healthcare.sample_collection]
+    en = Sample collection
+    ar = جمع العينات
+    be = Калекцыя ўзораў
+    bg = Събиране на проби
+    cs = Sběr vzorků
+    da = Prøveudtagning
+    de = Probenahme
+    el = Συλλογή δείγματος
+    es = Recogida de muestras
+    et = Proovide kogumine
+    eu = Lagin-bilketa
+    fa = مجموعه نمونه
+    fi = Näytteenotto
+    fr = Collecte d'échantillons
```

Comment:
fr = Prélèvements biologiques

Reply:
Thanks updated in fc7f0d210d023bef89b935a634311e2fc29019dc